mcp_zammad/server.py

```
@@ -956,9 +1302,9 @@ def get_ticket_resource(ticket_id: str) -> str:
                         ]
                     )
 
-                return "\n".join(lines)
+                return _truncate_response("\n".join(lines))
             except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)